Mathlib/Algebra/Bounds.lean

```diff
@@ -135,6 +152,25 @@ theorem BddBelow.mul {s t : Set M} (hs : BddBelow s) (ht : BddBelow t) : BddBelo
 #align bdd_below.mul BddBelow.mul
 #align bdd_below.add BddBelow.add
 
+@[to_additive]
+lemma BddAbove.range_mul {α : Type*} {f g : α → M} (hf : BddAbove (Set.range f))
```

Comment:
Hopefully we should be able to make a proof of this analogous to the one I suggested for `inv` using lemmas added in #9690

Reply:
Good idea; I'll make the change as soon as it's done merging.

Reply:
It's done now.